CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.2.5] - 2020-02-01
+## [1.0.0] - 2022-04-25
```

Comment:
Are we waiting to release it or doing it today?

```suggestion
## [1.0.0] - 2022-04-21
```

Reply:
yeah wasn't sure when we'd merge it so I put Monday